doc/content/reference/interop-repository/_index.md

```diff
@@ -1,27 +1,31 @@
 ---
-title: "Interoperability Repository"
+title: "LoRaWAN Backend Interfaces interoperability"
 description: ""
 ---
 
-{{% tts %}} expects the root of interoperability repository to contain `config.yml`, which contains various interoperablity configuration options.
+{{% tts %}} Network Server and Application Server can use LoRaWAN Backend Interfaces 1.0 and 1.1 with LoRaWAN Join Servers.
+
+<!-- more -->
+
+{{% tts %}} reads configuration from a file system. The root should contain `config.yml`, which contains various Join Servers.
 
 <!--more-->
 
 ```yml
 join-servers:                 # list of Join Server interoperability configurations,
                               # used to map a JoinEUI to the Join Server
-  - file: './path/js.yml'    # relative path to a file containing Join Server configiration
+  - file: './path/js.yml'     # relative path to a file containing Join Server configiration
```

Comment:
```suggestion
  - file: './path/js.yml'     # relative path to a file containing Join Server configuration
```

Reply:
This is an alignment issue on GitHub due to the `+` and `-` of the diff